PittAPI/people.py

```diff
@@ -20,61 +20,52 @@
 import subprocess
 
 import requests
+import grequests
 from bs4 import BeautifulSoup
-from requests.packages.urllib3.exceptions import InsecureRequestWarning
+from requests.packages.urllib3.exceptions import InsecureRequestWarning, InsecurePlatformWarning
 
 requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
-
+requests.packages.urllib3.disable_warnings(InsecurePlatformWarning)
 session = requests.session()
 
 
 def get_person(query, max_people=10):
-    '''
-    Doesn't work completely for now. IT WORKS
-    Returns a dict with URLs of user profiles. No scraping yet.
-    '''
 
     query = query.replace(' ', '+')
-    persons_list = []
-    tens = 0
-    while tens < max_people:
+    url_list = []
+    for i in range(int((max_people+9)/10)): #could replace with math.ceil, but this works and avoids importing math for one thing. int() cast for python 3
+
         url = "https://136.142.34.69/people/search?search=Search&filter={}&_region=kgoui_Rcontent_I0_Rcontent_I0_Ritems"
```

Comment:
See if you could possibly replace all of this curl junk with a requests call as well, not critical for me to merge the PR though